.lintstagedrc.yaml

```diff
@@ -0,0 +1,9 @@
+"*.ts":
+  # typescript
+  # prettier で、ダブルクオートからシングルクオートへ
+  - prettier --write --single-quote
+  - git add
```

Comment:
実行すると下記の警告が出ますが無視してよいのでしょうか?

> ⚠ Some of your tasks use `git add` command. Please remove it from the config since all modifications made by tasks will be automatically added to the git commit index.


```
$ git commit

> node-web-gpio@1.0.10 lint
> eslint index.ts


> node-web-gpio@1.0.10 precommit
> lint-staged

⚠ Some of your tasks use `git add` command. Please remove it from the config since all modifications made by tasks will be automatically added to the git commit index.


```